### PR TITLE
Catalog org list

### PR DESCRIPTION
**Pull request by Mussuranae**
Add the org list and the org view on catalog marketplace

### CODE REVIEW

File: apps/catalog/catalog/src/app/marketplace/organization/title/title.component.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<h1>Line-up</h1>
```
Comment (fritzschoff) [inline apps/catalog/catalog/src/app/marketplace/organization/title/title.component.html:1]: that should be library

File: apps/catalog/catalog/src/app/marketplace/organization/view/view.component.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<ng-container *ngIf="org$ | async as org">
+  <org-view [org]="org" [navLinks]="navLinks">
+    <ng-container *ngIf="org.email">
+      <a mat-stroked-button color="primary"
+        [href]="'mailto:' + org.email + '?subject=Meeting%20request%20via%20Archipel%20Market'">
+        <span>Ask for a meeting</span>
```
Comment (fritzschoff) [inline apps/catalog/catalog/src/app/marketplace/organization/view/view.component.html:6]: can we have meetings on AC? don't think so

File: apps/catalog/catalog/src/app/marketplace/organization/view/view.component.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import { Component, ChangeDetectionStrategy } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { pluck, switchMap } from 'rxjs/operators';
+import { OrganizationService } from '@blockframes/organization/+state/organization.service';
+
+const navLinks = [{
+  path: 'title',
+  label: 'Line-up'
```
Comment (fritzschoff) [inline apps/catalog/catalog/src/app/marketplace/organization/view/view.component.ts:8]: Library

**Review by fritzschoff — APPROVED** [review]

File: apps/catalog/catalog/src/app/marketplace/organization/title/title.component.html (hunk 1 above)
Comment (Mussuranae) [reply apps/catalog/catalog/src/app/marketplace/organization/title/title.component.html:1]: Vincent said that we can let as it is.

File: apps/catalog/catalog/src/app/marketplace/organization/view/view.component.html (hunk 2 above)
Comment (Mussuranae) [reply apps/catalog/catalog/src/app/marketplace/organization/view/view.component.html:6]: You're right, after asking to Vincent and Christophe, they said I can remove it.

**Pull request merged**